src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const cleanedSourceString = sourceString.replace(/\s*;\s*/g, ';').trim();
```

Comment:
The regular expression used in `replace` might not correctly handle cases where a semicolon is used within a CSS value, such as in a content property (e.g., `content: ";"`). It would be safer to split by semicolons that are followed by a newline or the end of the string.